mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -237,6 +237,30 @@ public boolean onNavigationItemSelected(MenuItem item) {
         return true;
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        String currentFragment = getSupportFragmentManager().findFragmentById(R.id.container)
+                .getClass()
+                .getSimpleName();
```

Comment:
This can be included in the previous line.

Reply:
That will create a style error like this 'the line exceeds 100 characters'